### PR TITLE
exiftool: 10.55 -> 10.60

### PR DESCRIPTION
**Pull request by kiloreux**
###### Things done

- [ ] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [x] Linux
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/top-level/perl-packages.nix [hunk 1]
```diff
@@ -7245,7 +7245,7 @@ let self = _self // overrides; _self = with self; {
 
       license = with licenses; [ gpl1Plus /* or */ artistic2 ];
 
-      maintainers = [ ];
+      maintainers = [ kiloreux ];
```
Comment (fpletz) [inline pkgs/top-level/perl-packages.nix]: Your maintainer entry is not in scope here like this: https://travis-ci.org/NixOS/nixpkgs/jobs/264326247#L548

It is in `lib.maintainers`, so as we're in a `with stdenv.lib;` you could either use `[ maintainers.kiloreux ]` or `with maintainers; [ kiloreux ]`.

**Review by fpletz — COMMENTED** [review]
The title of the commit should be: `exiftool: 10.55 -> 10.60`

**Comment by kiloreux** [comment]
@fpletz Implemented your comments. Feel free to merge them or ask for other changes.

**Comment by bjornfor** [comment]
@kiloreux: It seem the _commit_ still lacks "exiftool:" prefix. Amend the commit and force push.

**Comment by kiloreux** [comment]
@bjornfor not sure what you mean ?

**Comment by joachifm** [comment]
Changing the PR title does not affect the commit subject.

**Comment by kiloreux** [comment]
I am not quite sure how critical is the change of commit message @joachifm but I will be happy to change it if you see it's something I should do :smile:

**Comment by joachifm** [comment]
The point of the policy is to make it easy to identify package updates in the commit log. Please refer to the contribution guidelines for more information about commit messages.

**Comment by bjornfor** [comment]
Applied to master (c3acd48763ba47277407f3d477f7f959f59fcb2b "perlPackages.ImageExifTool: 10.55 -> 10.60").

**Pull request closed without merging**